docs/ops_guide/repush.md

```diff
@@ -85,4 +92,9 @@ and reducers. This is an experimental functionality that has not been vetted for
 it defaults to false.
 ```
 kafka.input.combiner.enabled = true
-```
\ No newline at end of file
+```
+## Future Work
+Repush will eventually support a more efficient way to perform [TTL](../user_guide/ttl.md).
```

Comment:
Let's give `ttl.md` a permalink too and link to that instead. Jekyll does not resolve markdown references. Docs for `Venice Recommended Development Workflow` and `Venice Quickstart` have some examples